spdx.info.yml

```diff
@@ -1,10 +1,27 @@
 name: SPDX Licenses
 type: module
 description: 'A license management module.'
-core: 8.x
 package: Joinup
+
+core: 8.x
 php: 7.1
+
 dependencies:
   - drupal:link
   - drupal:text
   - rdf_entity:rdf_entity
+
+config_devel:
```

Comment:
Is this a thing anymore? I thought that this entry was abandoned quite a while ago! How is this being used now?

Reply:
It is used for declaring and managing the configuration that is exported in a module: https://www.drupal.org/project/config_devel